release/util/sanity_check_windows.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# This script automatically download ray and run the sanity check for Windows
+# in various Python version. This script requires conda command to exist.
+
+unset RAY_ADDRESS
+export RAY_HASH="$RAY_HASH"
+export RAY_VERSION="$RAY_VERSION"
+
+if [[ -z "$RAY_HASH" ]]; then
+    echo "RAY_HASH env var should be provided"
+    exit 1
+fi
+
+if [[ -z "$RAY_VERSION" ]]; then
+    echo "RAY_VERSION env var should be provided"
+    exit 1
+fi
+
+if ! [ -x "$(command -v conda)" ]; then
```

Comment:
this should be `if [[ ! -x "$(command -v conda)" ]]; then`

Reply:
done